internal/ffmpeg/convert.go

```diff
@@ -78,6 +78,23 @@ func AvcConvertCommand(fileName, avcName, ffmpegBin, bitrate string, encoder Avc
 			avcName,
 		)
 
+	case VAAPIEncoder:
+		format := "format=nv12,hwupload"
+		result = exec.Command(
+			ffmpegBin,
+			"-hwaccel", "vaapi",
+			"-i", fileName,
+			"-c:a", "aac",
+			"-vf", format,
```

Comment:
@lastzero Unbelievable but I commited a mistake here. Line 88 should be `"-vf", format=nv12,hwupload ` instead.
Not specifying the format throws an error message when trying to encode. Could you correct this small mistake?

Reply:
I'm not quite sure I understand? "format" is a variable, so it is not the actual value of the parameter:
```golang
format := "format=nv12,hwupload"
```

Feel free to send us a quick pull request if you want to change any details (after you have tested it...)!

Reply:
Ooops, I posted quicker than I read.

I just got error messages on my system when trying to encode with VA-API and first thought it was about a missing specification of the format.
Now I see in line 82 I specified it. Sorry... my bad.

The error messages by the way are related to HEVC which means. The old Haswell architecture is not able to encode HEVC (so basically any new iPhone shot) video: https://wiki.libav.org/Hardware/vaapi

With H.264 however, it works with the snippet above.

Maybe I should accept that my NAS is simply too old for the new fancy bancy stuff. ;-)

Have a nice evening and sorry for the confusion!